packages/types/src/chip/index.ts

```diff
@@ -111,16 +123,15 @@ export type GetLeaderboardEntryRequest = z.infer<
 
 export const LeaderboardEntrySchema = z.object({
   username: z.string(),
-  chipIssuer: z.string(),
-  tapCount: z.number(),
-  // In the future could imagine name, chip fields being added to allow custom leaderboards corresponding to specific chips
+  entryValue: z.coerce.number(),
```

Comment:
I believe this is missing fields?

Reply:
i dont think you need the chipIssuer/entryType for this, since it is needed in every request you will know it on the frontend anyways